app/models/loan.rb

```diff
@@ -211,4 +211,24 @@ def healthy?
   def health_status_available?
     return !health_check.nil?
   end
+
+  def sum_of_disbursements(start_date: nil, end_date: nil)
+    return nil if transactions.empty?
+    transactions.by_type("disbursement").in_date_range(start_date, end_date).map { |t| t.amount }.sum
+  end
+
+  def sum_of_repayments(start_date: nil, end_date: nil)
+    return nil if transactions.empty?
+    transactions.by_type("repayment").in_date_range(start_date, end_date).map { |t| t.amount }.sum
+  end
+
+  def change_in_interest(start_date: nil, end_date: nil)
+    return nil if transactions.empty?
+    transactions.in_date_range(start_date, end_date).map { |t| (t.change_in_interest || 0) }.sum
```

Comment:
With manual testing, I found there are transactions with null change_in_interest (which err'ed on `sum`). I need to figure out why t.change_in_interest can be null sometimes. Either it should not be null or if there are transactions with null change_in_interest, should the this method return null?